src/server/process_request.c

```diff
@@ -1129,6 +1137,97 @@ dispatch_request(int sfds, struct batch_request *request)
 	return;
 }
 
+/**
+ * @brief
+ *	batch request for log
+ *
+ * @param[in] request - pointer to batch_request structure
+ * @param[in] stream  - connection stream
+ *
+ * @return Void
+ *
+ */
+
+static void
+log_request(struct batch_request *request, int stream)
+{
+	log_eventf(PBSEVENT_DEBUG2, PBS_EVENTCLASS_REQUEST, LOG_DEBUG, "",
```

Comment:
Updated to PBSEVENT_DEBUG

Reply:
When I said higher log level, I meant DEBUG3 or DEBUG4.  Something that isn't on at the default log level.

Reply:
We log batch requests in PBSEVENT_DEBUG which would be helpful in debugging issues. This will be also helpful in debugging as we can know what request had been last processed by the server.

Reply:
I'm not saying to remove it.  I'm just saying it doesn't sound like something that should be logged at the default log level.  We need to be careful when we add prolific messages to the default log level.  How prolific will this be?

Reply:
This will happen only when move and run or any inter-server communication happens. Which would be rare.

Reply:
To me this has the same feel has the 'Type XX received' message which is not printed at the default log level.  Maybe put it at the same level as the 'Type XX received' message.

Reply:
They are at the same level - DEBUG2.